README.rst

```diff
@@ -101,12 +101,24 @@ To enable this plugin:
     ``"twisted.internet.asyncioreactor.AsyncioSelectorReactor"``.
 
     .. note:: On existing projects that were not using the asyncio Twisted
-        reactor and make use of advanced Scrapy features, like interacting with
-        the Scrapy engine at run time, switching reactors may require
-        additional changes. If that is your case, see
-        `asyncio <https://docs.scrapy.org/en/latest/topics/asyncio.html>`_ in
-        the Scrapy documentation, specially `Awaiting on Deferreds
-        <https://docs.scrapy.org/en/latest/topics/asyncio.html#awaiting-on-deferreds>`_.
+        reactor, your existing code may need changes, such as:
+
+        -   Moving some module-level Twisted imports to the method or function
+            definitions where they are used, or `installing the asyncio reactor
+            <https://docs.scrapy.org/en/latest/topics/asyncio.html#installing-the-asyncio-reactor>`_
+            before those imports happen.
+
+            Some Twisted imports install the default, non-async Twisted
+            reactor as a side effect. Once a reactor is installed, it cannot be
+            changed for the whole run time.
```

Comment:
I think we should move/add this to Scrapy docs (cc @wRAR - we briefly discussed it, right?)

Reply:
https://github.com/scrapy/scrapy/issues/5981

Reply:
https://github.com/scrapy/scrapy/pull/6000.

I don’t think it should block the Scrapy 2.10 release (we can always link to master docs), but it may be best to hold on this PR until merged, to link to it.